t/31-client-obsolete.t

```diff
@@ -0,0 +1,34 @@
+# Copyright 2014-2025 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+use Mojo::Base -signatures;
+use Test::Most;
+use Test::Warnings ':report_warnings';
```

Comment:
```suggestion
```

Reply:
why do you want to remove this?

Reply:
Because it's never used in this test

Reply:
`:report_warnings` enables the feature to report all received unexpected warnings at the end of the test. So this line uses the feature, so it is used in the test. We use it in every test.